Preparedness/Preparedness/Views/CardView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  CardView.swift
+//  Preparedness
+//
+//  Created by Sebastian Wild on 10/25/20.
+//  Copyright © 2020 CCDC. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+struct CardView<Content: View>: View {
+    let headerText: String
+    @State var expanded: Bool = true
```

Comment:
super nit: this can be an enum driven layout instead of boolean

* `CardViewCompactType`
* `CardViewExpandedType`

Reply:
^ in case we ever wanted to expand to new layouts in the future

Reply:
Maybe - but I think for now a boolean suffices. The nice part about it is we can simply call `.toggle()` on tap. With an enum this won't be as easy (not that I can think of right now anyway). Still I think even if we stay with the `bool` for now it'll be trivial to change later on.